assets/src/components/billing/PricingOverview.tsx

```diff
@@ -144,9 +144,11 @@ export const PricingOptions = () => {
             </Paragraph>
 
             <Paragraph>
-              <li>5 seats included</li>
-              <li>Unlimited messages</li>
-              <li>Additional integrations</li>
+              <li>10 seats included</li>
+              <li>Slack integration</li>
+              <li>Unlimited data retention</li>
+              <li>Website screen sharing</li>
+              <li>Private Notes</li>
```

Comment:
"Notes" probably doesn't need to be capitalized, looks odd compared to everything above